Media/doc/introduction.md

```diff
@@ -0,0 +1,10 @@
+# Media Data Models
+
+These data models describe the main entities involved in multimedia sensor basing on computer vision. This pattern is based on using computer vision or speech analysis techniques to transform a device camera into an advanced sensor. Kurento provides, off-the-shelf, some basic computer vision capabilities for detecting faces, detecting crowds, detecting car plates, detecting movement, etc. Note that most of these analyses lead to generation of data that takes the form of events (e.g. detected a specific face, detected a car plate, detected an intruder, etc.)
```

Comment:
Per policy, we abstract Data Models from technologies. I.e. let's not mention Kurento, since other similar solutions may generate similar data.

Reply:
Done!